python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -250,17 +273,21 @@ def uniform_neighbor_sample(
             weight_t=weight_t,
             with_edge_properties=with_edge_properties,
             # FIXME accept and properly transmute a numpy/cupy random state.
-            random_state=hash(random_state, i),
```

Comment:
Just a comment that  catching bugs like this is why we need to get dask even on SG to run in CI ASAP. CC: @rlratzel